js/client/modules/@arangodb/testutils/testing.js

```diff
@@ -460,6 +297,43 @@ function loadTestSuites () {
   testFuncs['auto'] = autoTest;
 }
 
+let MODULES_LOADED=false;
+function loadModuleOptions () {
+  if (MODULES_LOADED) {
+    return false;
+  }
+  MODULES_LOADED = true;
+  const nonModules = ['clusterstats.js',
+                      'joinHelper.js',
+                      'testing.js',
+                      'block-cache-test-helper.js',
+                      'cluster-test-helper.js',
+                      'user-helper.js',
+                      'unittest.js'
+                     ];
+  let testModules = _.filter(fs.list(__dirname),
+                             function (p) {
+                               return (
+                                 p.substr(-3) === '.js' && !(
```

Comment:
```suggestion
                                 p.endsWith('.js') && !(
```

Reply:
this will make backports more complicated.